packages/frontend/src/components/dialogs/ViewGroup.tsx

```diff
@@ -445,9 +445,11 @@ function ViewGroupInner(
               className='group-separator'
             >
               {!isBroadcast
-                ? tx('n_members', group.contactIds.length.toString(), {
-                    quantity: group.contactIds.length,
-                  })
+                ? group.selfInGroup
+                  ? tx('n_members', group.contactIds.length.toString(), {
+                      quantity: group.contactIds.length,
+                    })
+                  : ''
```

Comment:
The element itself is still displayed in such a case, but it's fine since we'll rewrite this stuff anyway soon: https://github.com/deltachat/deltachat-desktop/issues/5707